tests/unit/Collectors/CustomCollectorTest.php

```diff
@@ -1,26 +1,47 @@
 <?php
 
-namespace AG\Tests\Collectors;
-
-use AG\ElasticApmLaravel\Collectors\TimelineDataCollector;
+use AG\ElasticApmLaravel\Agent;
+use AG\ElasticApmLaravel\Collectors\EventDataCollector;
 use Codeception\Test\Unit;
 use DMS\PHPUnitExtensions\ArraySubset\Assert;
+use Illuminate\Foundation\Application;
 use Illuminate\Support\Facades\Log;
 
-class TimelineDataCollectorTest extends Unit
+class CustomCollector extends EventDataCollector
```

Comment:
could also use an [anonymous class](https://www.php.net/manual/en/language.oop5.anonymous.php) instead.

Reply:
I thinks it's cleaner to have a separate class definition. Otherwise it will defined inside the `_before` method. feels a bit messy.

Reply:
🤷‍♂ fair enough.